packages/liferay-js-insights/insights/dependencies.js

```diff
@@ -0,0 +1,80 @@
+/**
+ * © 2019 Liferay, Inc. <https://liferay.com>
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+/**
+ * Extracts import declarations from a given AST applying the following transforms:
+ * - Relative imports are discarded
+ * - Imports are grouped per origin into:
+ * 	- clay3
+ *  - react
+ *  - js
+ *  - metal
+ *  - clay2
+ *  - others
+ */
+function extractImports(ast) {
+	const EMPTY = [[], [], [], [], [], []];
+	const SOURCE_GROUPS = [
+		source => source.includes('@clayui'),
+		source =>
+			source.includes('react') ||
+			source.includes('prop-types') ||
+			source.includes('classnames'),
+		source => source.includes('frontend-js-web'),
+		source => source.includes('metal-') || source === 'metal',
+		source => source.includes('clay-') || source === 'clay',
```

Comment:
Shouldn't all these `includes` be `startsWith`?

Reply:
yeah, probably... wondering if I did that on purpose or just copy-pasting...